src/components/TabSystem.js

```diff
@@ -98,6 +100,26 @@ const TabSystem = (props) => {
     });
   }
 
+  if (window.location.hash.length && !toastError.isActive("errorToast")) {
+    // eslint-disable-next-line no-restricted-globals
+    history.pushState("", "", window.location.origin);
+
+    toastError({
+      id: "errorToast",
+      duration: null,
+      render: () => (
+        <Alert status="error" variant="solid">
+          <AlertIcon />
+          <Box >
+            <AlertTitle>El plan compartido con el permalink, no pudo ser cargado correctamente</AlertTitle>
+            <AlertDescription>Si estas desde un celular probalo con la compu!</AlertDescription>
```

Comment:
Que pensas de este wording?

```suggestion
            <AlertTitle>El plan no se pudo cargar correctamente</AlertTitle>
            <AlertDescription>Si estás desde un celular, probalo en la compu!</AlertDescription>
```